src/jllvm/compiler/ClassObjectStubCodeGenerator.cpp

```diff
@@ -316,7 +291,8 @@ llvm::Function* jllvm::generateStaticCallStub(llvm::Module& module, const ClassO
 
     if (classObject.isUnintialized())
     {
-        buildClassInitializerInitStub(builder, classObject);
+        llvm::Value* classObjectLLVM = jllvm::classObjectGlobal(module, classObject.getDescriptor());
```

Comment:
```suggestion
        llvm::Value* classObjectLLVM = classObjectGlobal(module, classObject.getDescriptor());
```
Ditto in other places